src/tabs/presets/PresetTitlePanel.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+class PresetTitlePanel
+{
+    constructor(presetListDiv, preset, presetIndex, onLoadedCallback)
+    {
+        this._presetListDiv = presetListDiv;
+        this._index = presetIndex;
+        this._onLoadedCallback = onLoadedCallback;
+        this._domId = `preset_title_panel_${presetIndex}`;
+        this._preset = preset;
+
+        this._presetListDiv.append(`<div id="${this._domId}"></div>`);
+        this._domWrapperDiv = $(`#${this._domId}`);
+
+        this._domWrapperDiv.load("./tabs/presets/PresetTitlePanelBody.html", () =>
+        {
+            this._setupHtml();
+        });
+    }
+
+    subscribeClick(presetsDetailedDialog)
+    {
+        this._domWrapperDiv.on("click", () => {
```

Comment:
Single statement can be a oneliner:
```suggestion
        this._domWrapperDiv.on("click", () => presetsDetailedDialog.open(this._preset));
```